configs/app/features/safe.ts

```diff
@@ -0,0 +1,54 @@
+import type { Feature } from './types';
+
+import chain from '../chain';
+
+// https://docs.safe.global/safe-core-api/available-services
+const SAFE_API_MAP: Record<string, string> = {
```

Comment:
Since it is ad hoc solution, I don't think that it is really necessary. Eventually, we hope to get this info from our API, so in case of passing config through ENV variable migration to permanent solution will be more painful.